transfer/requirements.md

```diff
@@ -62,14 +65,17 @@ It may be desirable to bundle these notifications in a daily digest, rather than
 
 #### For Machines
 
-Notifications indicating an error has occurred will have a `code` value of `400`. The `error` value indicates a general category of error, while the `message` value provides information about the specific error that occurred. The optional `action` value indicates system actions that have been taken as a result of the error and `time` is a timestamp of the time the error occurred.
+Notifications indicating an error has occurred will have a `type` value of `Reject`. The `summary` provides information about the specific error. The `content` value provides specific information about reason for the error. The `object` value provides a reference to the object the action was taken on. The optional `result` value indicates system actions that have been taken as a result of the error and `endTime` is a timestamp of the time the error occurred.
 
 ```
 {
-  "code": 400,
-  "error": "Bag validation failure",
-  "message": "Metadata elements are missing",
-  "action": "Staged for deletion",
+  "type": "Reject",
+  "summary": "Bag validation failure",
+  "content": "Metadata elements are missing",
```

Comment:
Let's get rid of `content` and put that text in `summary`. On further reflection I'm not sure it makes sense to create a hierarchy of error messages.